src/components/my-history/custom-my-history.vue

```diff
@@ -38,15 +38,7 @@
         </template>
       </MyHistory>
 
-      <div v-else class="x-flex x-flex-1 x-flex-col x-items-center">
-        <NoHistoryIcon
-          class="x-flex-no-shrink"
-          :class="{ 'x-grayscale': !$x.isHistoryQueriesEnabled }"
-        />
-        <p class="x-title3 x-py-16 x-text-neutral-75">
-          {{ $t('myHistory.noHistory') }}
-        </p>
-      </div>
+      <div v-else />
```

Comment:
Why are we assuming that we have to delete the My History graphic based on the designs?

https://www.figma.com/file/mYfJZHn668mGEGf61Sji6m/X-Archetype?node-id=3027%3A22233&mode=dev

There are two versions in there..